skops/card/_model_card.py

```diff
@@ -375,6 +376,63 @@ def add_metrics(self, **kwargs: str) -> "Card":
             self._eval_results[metric] = value
         return self
 
+    def add_permutation_importances(
+        self,
+        permutation_importances,
+        columns,
+        plot_file="permutation_importances.png",
+        plot_name="Permutation Importances",
+        overwrite=False,
+    ) -> "Card":
+        """Plots permutation importance and saves it to model card.
+
+        Parameters
+        ----------
+        permutation_importances : sklearn.utils.Bunch
+            Output of :func:`sklearn.inspection.permutation_importance`.
+
+        columns : str, list or pandas.Index
+            Column names of the data used to generate importances.
+
+        plot_file : str
+            Filename for the plot.
+
+        plot_name : str, or Path
+            Name of the plot.
+
+        overwrite : bool
+            Whether to overwrite the permutation importance plot, if exists.
+
+        Returns
+        -------
+        self : object
+            Card object.
+        """
+        try:
+            import matplotlib.pyplot as plt
+        except metadata.PackageNotFoundError:
+            raise ModuleNotFoundError(
+                "This feature requires matplotlib to be installed."
+            )
+        if str(plot_file) in self.render() and overwrite is False:
```

Comment:
Could you please explain why we need to check `str(plot_file) in self.render()`?

Reply:
none of the Card attributes point to the path to the repository where we could look for plot (or even if there's a plot file, it might not be in the model card) so I'm checking if it exists in the card itself. if so, we have to overwrite.

Reply:
Can we not check if `plot_file` exists on the file system, so basically `if Path(plot_file).exists() and not overwrite: ...`? Or am I missing something?